sansa-query-spark/src/test/resources/net/sansa_stack/query/spark/dof/answers_query0_0.txt

```diff
@@ -0,0 +1,19 @@
+X Y Z
```

Comment:
Please, remove all the files which aren't needed (which can be generated when running uni-tests and removed afterward).

Reply:
`answers_*` are used as testing etalons. they are not generated by uni tests but used for checking the correctness of the results